alembic/env.py

```diff
@@ -21,7 +21,11 @@
 import sys
 
 sys.path.append(os.getcwd())
-config.set_main_option('sqlalchemy.url', os.getenv('DEV_DATABASE_URL'))
+config.set_main_option(
+    'sqlalchemy.url',
+    os.getenv('TEST_DATABASE_URL') if os.getenv('APP_SETTINGS') == 'testing'
+    else os.getenv('DEV_DATABASE_URL')
```

Comment:
I think this will make us use Dev_database even on production. We should also check for the production settings to get the production database.

Reply:
I also think this is better setup on the config file.

Reply:
> I think this will make us use Dev_database even on production. We should also check for the production settings to get the production database.

I too had similar thoughts regarding this but @philophilo mentioned that migrations are run differently on production. I plan on asking for further clarity however given the error posted on slack over the weekend

Reply:
> I also think this is better setup on the config file.

The config file would not suit this seeing as the configuration being done here is in regards to alembic which,  if one goes through the documents (https://alembic.sqlalchemy.org/en/latest/api/config.html), requires this to be put in the env.py

Reply:
> > I think this will make us use Dev_database even on production. We should also check for the production settings to get the production database.
> 
> I too had similar thoughts regarding this but @philophilo mentioned that migrations are run differently on production. I plan on asking for further clarity however given the error posted on slack over the weekend

@RonKbS  any update from @philophilo regarding this?

Reply:
We are to have a sync later today as he was unavailable in the morning.

Reply:
Okay.

Reply:
Following the sync with @philophilo, I have updated this to include a check for production settings as well.